### PR TITLE
article type component

### LINKED ISSUE #79: Implement Desgins on "What do you want to submit?"
See https://projects.invisionapp.com/share/5EUTER8KBH3#/screens/393186076

Drop downs of article type
This changes text below it based on descriptive text for that article type.

### PR DESCRIPTION
**Pull request by will-byrne**
Closes #79 

Added the ArticleType component and altered the SelectField to allow a default value.

### CODE REVIEW

File: src/initial-submission/components/ArticleType.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+            </Paragraph>
+            <Paragraph type="writing" secondary>
+                {t('article-types:feature-article.paragraph-2')}
+            </Paragraph>
+            <Paragraph type="writing" secondary>
+                {t('article-types:feature-article.paragraph-3')}
+                <a
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    href="https://reviewer.elifesciences.org/author-guide/editorial-process"
```
Comment (Cooryd) [inline src/initial-submission/components/ArticleType.tsx]: Should these be relative links?

File: src/initial-submission/components/ArticleType.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+import { useTranslation } from 'react-i18next';
+import { ValueType } from 'react-select/src/types';
+import { SelectField, Paragraph } from '../../ui/atoms';
+import { Value } from '../../ui/atoms/SelectField';
+
+const ArticleType = (): JSX.Element => {
+    const { t } = useTranslation();
+    const articleTypes = [
+        {
+            label: 'Research Article',
```
Comment (Cooryd) [inline src/initial-submission/components/ArticleType.tsx]: Should these labels be translation keys?

**Comment by diversemix** [comment]
As this is our "first" page as such ... dunno if @davidcmoulton wants to take a look - re: how we are doing language / accessibility?

File: src/initial-submission/components/ArticleType.tsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    const changeArticleType = (value: ValueType<Value>): void => {
+        const newArticleType: string = (value as Value).value;
+
+        if (descriptionTypeMap[newArticleType]) {
+            setDescription(descriptionTypeMap[newArticleType]());
+        }
+    };
+    return (
+        <div>
+            <h1 className="typography__heading typography__heading--h1">What would you like to submit?</h1>
```
Comment (Cooryd) [inline src/initial-submission/components/ArticleType.tsx]: Should this be a translation?

**Pull request merged**